packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "invalidObject": {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column",
+    alignItems: "center",
+    textAlign: "center",
+    h1: { marginBottom: "30px" },
+    svg: { marginBottom: "30px" },
+    button: { fontSize: "20px" }
+  },
+  "invalidObjectSorryText": { maxWidth: "70%" },
+})
+
+export interface InvalidObjectProps {
+  isBeta?: string
```

Comment:
This prop isn't showing up the documentation site.  You need to add a description for it to show up.  Also seems like this should be a boolean.  Maybe something like

```suggestion
  /** Indicates if an additional "/beta" string should be added to when navigating back to the home screen. */
  isBeta?: boolean
```